learn/polkadot/polkadot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Tanssi and Polkadot
+description: Introduction to polkadot
+---
+
+# Tanssi and Polkadot
```

Comment:
This page seems to be about Polkadot and not really about Tanssi with the exception of one sentence. Just food for thought. Not sure about this title 🤷‍♀️